tests/checkpoint/test_checkpoint.py

```diff
@@ -5134,3 +5141,142 @@ def test_newstyle_checkpoint_result_validations_include_rendered_content(
         .rendered_content
         is not None
     )
+
+
+@pytest.mark.integration
+@pytest.mark.cloud
+@pytest.mark.parametrize(
+    "checkpoint_config,expected_validation_id",
+    [
+        pytest.param(
+            CheckpointConfig(
+                name="my_checkpoint",
+                config_version=1,
+                run_name_template="%Y-%M-foo-bar-template",
+                expectation_suite_name="my_expectation_suite",
+                action_list=[
+                    {
+                        "name": "store_validation_result",
+                        "action": {
+                            "class_name": "StoreValidationResultAction",
+                        },
+                    },
+                ],
+                validations=[
+                    CheckpointValidationConfig(
+                        batch_request={
+                            "datasource_name": "my_datasource",
+                            "data_connector_name": "my_basic_data_connector",
+                            "data_asset_name": "Titanic_1911",
+                        },
+                    ),
+                ],
+            ),
+            None,
+            id="no ids",
+        ),
+        pytest.param(
+            CheckpointConfig(
+                name="my_checkpoint",
+                config_version=1,
+                default_validation_id="7e2bb5c9-cdbe-4c7a-9b2b-97192c55c95b",
+                run_name_template="%Y-%M-foo-bar-template",
+                expectation_suite_name="my_expectation_suite",
+                batch_request={
+                    "datasource_name": "my_datasource",
+                    "data_connector_name": "my_basic_data_connector",
+                    "data_asset_name": "Titanic_1911",
+                },
+                action_list=[
+                    {
+                        "name": "store_validation_result",
+                        "action": {
+                            "class_name": "StoreValidationResultAction",
+                        },
+                    },
+                ],
+                validations=[],
+            ),
+            "7e2bb5c9-cdbe-4c7a-9b2b-97192c55c95b",
```

Comment:
i'm curious, what's this UUID?

Reply:
I just used `uuigen` for a random value - does this not adhere to the structure of cloud ids? I can change if need be.